src/Data/String/Extra.purs

```diff
@@ -160,7 +160,7 @@ regexUnicodeWords =
   rsDingbat = "[" <> rsDingbatRange <> "]"
   rsLower = "[" <> rsLowerRange <> "]"
   rsMisc = "[^" <> rsAstralRange <> rsBreakRange <> rsDigit <> rsDingbatRange <> rsLowerRange <> rsUpperRange <> "]"
-  rsFitz = "\\ud83c[\\udffb-\\udfff]"
+  -- rsFitz = "\\ud83c[\\udffb-\\udfff]"
```

Comment:
I'm not sure why this is here, nor whether it was previously used somewhere

Reply:
https://github.com/lodash/lodash/blob/master/.internal/unicodeWords.js#L27

Reply:
Gotcha. Thanks @toastal!